packages/cojson/src/tests/permissions.test.ts

```diff
@@ -2913,6 +2914,70 @@ test("revoking access on a parent group doesn't block access to the child group
   expect(mapOnNode2.get("foo")).toEqual("baz");
 });
 
+test("revoking access to parent group", async () => {
```

Comment:
We also need tests to verify that:
- After revokeExtend the account loses read access over the new transactions (key rotation)
- When revoking the extend of a granparent the related members lose read access on the grandchilds (key rotation on childs)